package.json

```diff
@@ -19,29 +19,22 @@
     "@apollo/client": "^3.3.6",
     "@azure/functions": "^1.2.2",
     "@definitelytyped/header-parser": "^0.0.64",
-    "@graphql-codegen/fragment-matcher": "^1.15.4",
     "@octokit/graphql-schema": "^10.11.0",
     "@octokit/webhooks": "^7.21.0",
     "@types/node": "latest",
     "@types/node-fetch": "^2.5.7",
     "@types/prettyjson": "^0.0.29",
-    "@types/request": "^2.48.5",
-    "chalk": "^4.1.0",
     "dayjs": "^1.10.3",
     "fast-json-patch": "^3.0.0-1",
     "fs-extra": "^9.0.1",
-    "graceful-fs": "^4.2.4",
     "graphql": "^14.5.8",
-    "idembot": "^0.0.12",
     "node-fetch": "^2.6.1",
     "prettyjson": "^1.2.1",
-    "request": "^2.88.2",
-    "tslib": "^2.0.3",
     "typescript": "^4.1.3",
     "yargs": "^16.2.0"
   },
   "devDependencies": {
-    "@endemolshinegroup/cosmiconfig-typescript-loader": "^3.0.2",
```

Comment:
This was added recently, in #317 https://github.com/DefinitelyTyped/dt-mergebot/pull/317/commits/ae68ccb7f40cf8da7c9179f75bafa3412c73eb12#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R44, maybe by accident?

Reply:
That was very intentional, it was my solution to a version mess without it.  To see it:

    rm -rf node_modules package-lock.json; npm i

Reply:
Ahh, I see. But it seems to be resolved now?
```Shell
$ git checkout upstream/master 
HEAD is now at 0f7f9425 Call Day.js directly
$ rm-rf node_modules package-lock.json
$ npm install
npm WARN deprecated @types/commander@2.12.2: This is a stub types definition for commander (https://github.com/tj/commander.js). commander provides its own type definitions, so you don't need @types/commander installed!
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

added 1122 packages, and audited 1122 packages in 2m

69 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Reply:
Are you sure that you tested from this PR's branch?  (The command that you have on top checks out `upstream/master`.)

Here's what I'm getting with that one removed:

```
mustafa:~/dt/bot eli> rm -rf node_modules package-lock.json; npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: ts-node@8.10.2
npm ERR! node_modules/ts-node
npm ERR!   ts-node@"^8" from @endemolshinegroup/cosmiconfig-typescript-loader@1.0.2
npm ERR!   node_modules/@endemolshinegroup/cosmiconfig-typescript-loader
npm ERR!     @endemolshinegroup/cosmiconfig-typescript-loader@"^1.0.0" from apollo-language-server@1.25.0
npm ERR!     node_modules/apollo-language-server
npm ERR!       apollo-language-server@"^1.25.0" from apollo@2.32.1
npm ERR!       node_modules/apollo
npm ERR!         dev apollo@"^2.32.0" from the root project
npm ERR!       1 more (apollo-codegen-core)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional ts-node@">=9.0.0" from jest-config@26.6.3
npm ERR! node_modules/jest-config
npm ERR!   jest-config@"^26.6.3" from @jest/core@26.6.3
npm ERR!   node_modules/@jest/core
npm ERR!     @jest/core@"^26.6.3" from jest@26.6.3
npm ERR!     node_modules/jest
npm ERR!       dev jest@"^26.4.4" from the root project
npm ERR!       1 more (ts-jest)
npm ERR!     1 more (jest-cli)
npm ERR!   jest-config@"^26.6.3" from jest-cli@26.6.3
npm ERR!   node_modules/jest-cli
npm ERR!     jest-cli@"^26.6.3" from jest@26.6.3
npm ERR!     node_modules/jest
npm ERR!       dev jest@"^26.4.4" from the root project
npm ERR!       1 more (ts-jest)
npm ERR!   2 more (jest-runner, jest-runtime)
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/eli/.npm/cache/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/eli/.npm/cache/_logs/2021-01-19T00_08_58_171Z-debug.log
```

Reply:
Uhh that was dumb of me. Of course it's resolved in `master` -- thanks to the extra dependency we're discussing! I reproduced this error and restored the dependency. :heavy_check_mark: Sorry for the noise!